### ISSUE TITLE
The page build failed for the `master` branch with the following error

### ISSUE THREAD

**Issue by mgifford** [abhn/Elementary#5]
I got this error from Jekyll when pushing this to Github:

> The page build failed for the `master` branch with the following error: 
> 
> The submodule registered for `./elementary` could not be cloned. Make sure it's using https:// and that it's a public repo. For more information, see help.github.com/en/github/working-with-github-pages/troubleshooting-jekyll-build-errors-for-github-pages-sites#invalid-submodule. 
> 

I assume this is because "http://rubygems.org" is used rather than source "https://rubygems.org".

**Comment by abhn** [comment]
Yes, let's see if it comes up again. I'm hosting on netlify, so I don't see any errors/warnings. I'll pull your PR and check if I can fix that build failure error

**Comment by mgifford** [comment]
I've been looking around and one possible suggestion was modifying the .gitmodules to use https.  I can't see why this would be different. 

```
[submodule "elementary"]
        path = elementary
        url = https://github.com/abhn/Elementary.git
```

But it seemed to nudge me ahead to this error:

The page build failed for the `master` branch with the following error: 

Your SCSS file `elementary/assets/main.scss` has an error on line 1: File to import not found or unreadable: main. For more information, see help.github.com/en/github/working-with-github-pages/troubleshooting-jekyll-build-errors-for-github-pages-sites#invalid-sass-or-scss.

**Comment by abhn** [comment]
Aah, that's weird. Especially because it runs fine locally and on Netlify. Thank you for letting me know

**Comment by abhn** [comment]
Okay, so I discovered that this is also a way to do it: https://github.blog/2017-11-29-use-any-theme-with-github-pages/ basically just involves adding a `remote_theme: abhn/Elementary` to the config. It seems to get rid of the error, but then I'm not sure how is one supposed to run it locally. 

In my personal site, I've added both theme and remote_theme. Not ideal, but works.

**Comment by mgifford** [comment]
All these little things..  Thanks for circle back with this.
